chapters/curate-maintain/content/reading/README.md

```diff
@@ -1,25 +1,35 @@
 ### Content
 
-Owner: Adrian Șendroiu
-Reviewer: Ștefan Jumărea
+The content maintainer is responsible for overseeing different aspects of the open educational resource, like content management, accepting and reviewing contributions, or community building.
 
-the central part is content management, around a repository
+It is important for such a resource to clearly identify who the maintainer is, for the purpose of transparency and good communication.
 
-typically Git & GitHub
+The content is typically organized in a `Git` repository, which has a series of advantages like version control, branching, history, etc.
+It is also recommended to use a managed development platform like `GitHub` or `GitLab`, which is built on top of `Git`, while also providing additional useful features, like code review, issues, pull requests, integrations, and others.
 
-versioning, maintaining up-to-date
+The content should be published under a license.
+An example of a widely used license for public education resources is `Creative Commons`.
```

Comment:
I would rephrase this to state that we strongly encourage (and use) CC licencing